sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -3,479 +3,622 @@
+    options?: CreateSessionReceiverOptions
+  ): SessionReceiver<"peekLock"> & SubscriptionRuleManagement;
+  createSessionReceiver(
+    topicName: string,
+    subscriptionName: string,
+    receiveMode: "receiveAndDelete",
+    sessionId: string | "",
+    options?: CreateSessionReceiverOptions
+  ): SessionReceiver<"receiveAndDelete"> & SubscriptionRuleManagement;
 }
 
 // @public
 export interface ServiceBusClientOptions {
-    dataTransformer?: DataTransformer;
-    webSocketOptions?: WebSocketOptions;
+  dataTransformer?: DataTransformer;
+  webSocketOptions?: WebSocketOptions;
 }
 
-// Warning: (ae-forgotten-export) The symbol "ReceivedMessage" needs to be exported by the entry point index.d.ts
-//
-// @public
-export class ServiceBusMessage implements ReceivedMessage {
-    abandon(propertiesToModify?: {
-        [key: string]: any;
-    }): Promise<void>;
-    readonly _amqpMessage: AmqpMessage;
-    body: any;
-    clone(): SendableMessageInfo;
-    complete(): Promise<void>;
-    contentType?: string;
-    correlationId?: string | number | Buffer;
-    deadLetter(options?: DeadLetterOptions): Promise<void>;
-    readonly deadLetterSource?: string;
-    defer(propertiesToModify?: {
-        [key: string]: any;
-    }): Promise<void>;
-    readonly delivery: Delivery;
-    readonly deliveryCount?: number;
-    readonly enqueuedSequenceNumber?: number;
-    readonly enqueuedTimeUtc?: Date;
-    readonly expiresAtUtc?: Date;
-    get isSettled(): boolean;
-    label?: string;
-    lockedUntilUtc?: Date;
-    readonly lockToken?: string;
-    messageId?: string | number | Buffer;
-    partitionKey?: string;
-    replyTo?: string;
-    replyToSessionId?: string;
-    scheduledEnqueueTimeUtc?: Date;
-    readonly sequenceNumber?: Long;
-    sessionId?: string;
-    timeToLive?: number;
-    to?: string;
-    userProperties?: {
-        [key: string]: any;
-    };
-    viaPartitionKey?: string;
+// @public
+export class ServiceBusMessage implements ReceivedMessageInfo {
+  abandon(propertiesToModify?: { [key: string]: any }): Promise<void>;
+  readonly _amqpMessage: AmqpMessage;
+  body: any;
+  clone(): SendableMessageInfo;
+  complete(): Promise<void>;
+  contentType?: string;
+  correlationId?: string | number | Buffer;
+  deadLetter(options?: DeadLetterOptions): Promise<void>;
+  readonly deadLetterSource?: string;
+  defer(propertiesToModify?: { [key: string]: any }): Promise<void>;
+  readonly delivery: Delivery;
+  readonly deliveryCount?: number;
+  readonly enqueuedSequenceNumber?: number;
+  readonly enqueuedTimeUtc?: Date;
+  readonly expiresAtUtc?: Date;
+  get isSettled(): boolean;
+  label?: string;
+  lockedUntilUtc?: Date;
+  readonly lockToken?: string;
+  messageId?: string | number | Buffer;
+  partitionKey?: string;
+  replyTo?: string;
+  replyToSessionId?: string;
+  scheduledEnqueueTimeUtc?: Date;
+  readonly sequenceNumber?: Long;
+  sessionId?: string;
+  timeToLive?: number;
+  to?: string;
+  userProperties?: {
```

Comment:
Are we settled on this name `userProperties` , Java we are calling `properties`  ?

Reply:
Great question! @JoshLove-msft , @ShivangiReja  - I seem to recall you were involved a conversation about this for .NET as well. Where did you land?

Reply:
Yes, we settled on Properties.

Reply:
Renamed.